velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -32,7 +32,9 @@ void Writer::write(const RowVectorPtr& data) {
       recordBatch->schema(), recordBatch->columns(), data->size());
   if (!arrowWriter_) {
     stream_ = std::make_shared<DataBufferSink>(pool_);
-    auto arrowProperties = ::parquet::ArrowWriterProperties::Builder().build();
+    auto arrowProperties = ::parquet::ArrowWriterProperties::Builder()
```

Comment:
Can you pass in the `ArrowWriterProperties` in the constructor and enable int96 in the corresponding unit tests only?

Reply:
Passes `writeInt96AsTimestamp` through writer option.